feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md

```diff
@@ -52,19 +52,19 @@ Ensure that when **all LAG member** become set with forwarding-viable == FALSE.
   - Run traffic:
     - From prefix pfx1 to all three: pfx2, pfx3, pfx4
     - From prefix pfx2 to: pfx1
-  - Make the forwarding-viable transitions from TRUE --> FALSE on ports 3-7
+  - Make the forwarding-viable transitions from TRUE --> FALSE on ports 3-6
     within the LAG_2 on the DUT
     - ensure that only DUT port 2 of LAG ports has bidirectional traffic.
-    - Ensure there is no traffic transmitted out of DUT ports 3-7
-    - ensure that traffic is received on all port2-7 and delivered to ATE port1
+    - Ensure there is no traffic transmitted out of DUT ports 3-6
+    - ensure that traffic is received on all port2-6 and delivered to ATE port1
     - ensure there are no packet losses in steady state (no congestion).
     - Ensure there is no traffic received on DUT LAG_3
     - Ensure there is no traffic transmitted on DUT LAG_3
   - Disable/deactive laser on ATE port2; All LAG_2 members are either down (port2) or
     set with forwarding-viable=FALSE
     - Ensure ISIS adjacency is UP on DUT LAG_2 and ATE LAG_2
```

Comment:
This is outdated; the readme requires an update to match the latest spec (https://github.com/openconfig/public/blob/a2c7a09e619c56782b3f4495a065fd7c34ef1175/release/models/interfaces/openconfig-if-sdn-ext.yang#L69)

cc @rszarecki @sachendras @dplore

Reply:
Thanks @LimeHat for the review. I have updated the Readme file as well in same commit because of some changes in the test/topology and working to resolve another comment.

Reply:
> I have updated the Readme file as well in same commit because of some changes in the test/topology and working to resolve another comment.

To clarify, my comment is not about the changes in the topology you made (reducing the port count). After the readme was written by @rszarecki, there was a change in the definition of how the devices should behave when all ports are in a non-viable state.

Specifically, the current expectation is that IS-IS and BGP adjacencies will be dropped (either immediately or after timing out on hello/dead timers). Hence I don't think it makes much sense to merge a test that validates outdated behavior, and we need to update the README first to describe the new behavior together with the code.

Reply:
Thanks @LimeHat for the clarification, I agreed with you, But to clarify the expectation here is when all the link of LAG is set with non-viable and ISIS/BGP must be UP by updating the Hello/Dead interval and make sure traffic is only receiving from server and process it to client(As TX must be Only Blocked).

Reply:
> But to clarify the expectation here is when all the link of LAG is set with non-viable and ISIS/BGP must be UP by updating the Hello/Dead interval and make sure traffic is only receiving from server and process it to client(TX only must be Blocked).

In my view, the following things should occur:
1) tx is blocked immediately
2) ISIS and BGP time out after the timer expires, and after that, there will be no RX to ISIS/BGP prefixes through this LAG 

It makes sense to validate the IS-IS transition from the `up` to `down` state after the hold timer expires to make sure that the new behavior (don't transmit all L3 packets, including the control plane packets) works as expected. If you simply increase the hold timer to a large value, this behavior is not going to be checked.

Reply:
Hi @LimeHat 

I agreed with you the ISIS is getting down after timer expired. But i am confused with below:

 Readme says:
     Disable/deactivate laser on ATE port2; All LAG_2 members are either down (port2) or set with forwarding- 
     viable=FALSE
      -Ensure ISIS adjacency is UP on DUT LAG_2 and ATE LAG_2 (It is getting down immediately as setting forwarding- 
     viable=FALSE on all LAG ports)

If the above is the expected behavior then the README.md required to update with isis Down instead of isis UP in this case right?

Reply:
Yes, exactly; the readme was written before this behavior was standardized and hence requires an update.

Reply:
> Yes, exactly; the readme was written before this behavior was standardized and hence requires an update.

+1, the README must be updated to reflect the current requirement which says that the DUT must stop sending all L3 packets when forwarding_viable is false.

Suggested updates 
  - RT-5.7.1.2: Verify forwarding-viable behavior on an aggregate interface with all members down or set with forwarding-viable=FALSE.
    - Ensure ISIS adjacency is UP on DUT LAG_2 and ATE LAG_2
    - Disable/deactive laser on ATE port2;  Now all LAG_2 members are either down (port2) or
    set with forwarding-viable=FALSE
    - Ensure there is no layer3 traffic transmitted out of  DUT ports 2-6 (LAG_2)
    - Ensure that traffic is received on all port3-6 and delivered to ATE LAG_1
    - Ensure there are no packet losses in steady state (no congestion) for
      traffic from ATE LAG_2 to ATE LAG_1 (pfx_1).
    - Ensure there are no packet losses in steady state (no congestion) for
      traffic from ATE LAG_1 to ATE LAG_3 (pfx_2, pfx3).
    - Ensure there is no traffic received on DUT LAG_3
    - Ensure that traffic from ATE port1 to pfx2, pfx3 are transmitted via DUT
      LAG3
    - Ensure that traffic from ATE port1 to pfx4 are discarded on DUT
    - Ensure that the ISIS adjacency times out on DUT LAG_2 and ATE LAG_2

(note I moved the verify ISIS UP to occur before disabling the final remaining LAG member port)

Reply:
Thanks @dplore and @LimeHat for the response i will update the README as you suggested.

Reply:
Updated as Suggested